theano/gof/tests/test_op.py

```diff
@@ -233,6 +235,28 @@ def c_code(self, node, name, inputs, outputs, sub):
             self.assertRaises((NotImplementedError, utils.MethodNotDefined),
                               thunk)
 
+    def test_no_make_node(self):
+        class IncOne(Op):
+            """An Op without make_node"""
+            __props__ = ()
+            itypes = [T.fmatrix]
+            otypes = [T.fmatrix]
+
+            def perform(self, node, inputs, outputs):
+                input, = inputs
+                output, = outputs
+                output[0] = input + 1
+
+        x_input = T.fmatrix('x')
+        o = IncOne()(x_input)
+
+        # Confirming that make_node method is implemented
```

Comment:
@nouiz Earlier, since "make_node" method was compulsory to be implemented by all the class that inherits Op, I wrote a test that checks it's not compulsory any more if "itypes" and "otypes" are defined.